netbare-core/src/main/java/com/github/megatronking/netbare/net/UidDumper.java

```diff
@@ -29,6 +35,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
+import java.net.InetSocketAddress;
 import java.util.concurrent.Callable;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeUnit;
```

Comment:
Many/all imports in this file should no longer be required after having converted `UidDumper` to an interface